src/calculator.js

```diff
@@ -0,0 +1,84 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+
+const Button = (props) => {
+  const { label } = props;
+  return (
+    <button type="button">{label}</button>
+  );
+};
+
+const Screen = () => (
+  <div className="screen-row">
+    <input type="text" value={0} readOnly />
+  </div>
+);
```

Comment:
- [ ] Please make sure that all components are class-based as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/205930648-d40188f9-e24c-467b-8402-d55e5d378abf.png)